tests/js/common/aql/aql-view-arangosearch-feature.js

```diff
@@ -1618,6 +1618,141 @@ function iResearchFeatureAqlTestSuite () {
       }
     },
     
+    testCustomAqlAnalyzer : function() {
```

Comment:
This test looks more like a test suite with a tearDown removing the analyzer.
And each testBlock it's own test.

Reply:
As we have many analyzers to test - putting each case for each analyzer into separate suite will make tests run much longer as instantiating suite takes time. So we decided to keep analyzers in separate test methods and do cleanup in `finally` blocks to speed up test runs

Reply:
Isn't this the use-case for:
setupAll() << initialize the shared constant part
setup() << initiliaze the local test part
teardown() << drop local test part (what is now in finally)
teardownAll() << drop shared constant part